index.html

```diff
@@ -25,15 +26,203 @@ <h1> I’m George
                 Glad to see you!</h1>
         </section>
         
-    
+    <div class="primary-div">
         <p class="primary">I’m a software developer! I can help you
-                 build a product , feature or website Look
-                  through some of my work and 
-                  experience! If you like what you see and
-                   have a project you need coded, don’t 
-                   hestiate to contact me.</p>
+            build a product , feature or website Look
+             through some of my work and 
+             experience! If you like what you see and
+              have a project you need coded, don’t 
+              hestiate to contact me.</p>
+              <br>
+
+    <a href="#"class="connect">LET’S CONNECT </a>
+    </div>
+        <br><br><br>
+        
+        <div class="icons" > 
+            <ul class="socials-2">                
+                <li> <a href="https://twitter.com/geok8376"><img src="img/Vectortwit.png" class="twit"></a> </li>
+                <li> <a href="https://www.linkedin.com/in/george-kinyanjui-030468149/"><img src="img/Linkedin icon.png"  alt="linkedin"> </a>  </li>
+                <li> <a href="#"><img src="img/Vectorm.png" alt="m"></a> </li>
+                <li> <a href="https://github.com/george827"><img src="img/Vectorgit.png" alt="git"></a> </li>
+                <li> <a href="#"><img src="img/Vectorch.png" alt="icon"></a> </li>
+            </ul>
+        </div>
+    </section>
+
+    </section>
+
+    <section class="snapshot1">
+        <img src="img/Snapshoot-Portfolio1.png" alt="snapshot1" class="image-po">
+        <section id="main">
+
+            
+            <div class="head-one">
+                <h1>Tonic</h1>
+            </div>
+                <div class="grid-container">
+                    <div class="grid-item"> <span class="canop">CANOPY</span></div>
+                    <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> Back End Dev</div>
+                    <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> 2015</div>  
+                </div>
+                
+            
```

Comment:
- [ ] Good job so far 👏 I would suggest that you remove these empty spaces between your codes because they make your codes look scattered and disorganized which is not the best coding practice.